src/main/java/org/takes/facets/fork/FkEncoding.java

```diff
@@ -71,18 +71,18 @@
     private final transient String encoding;
 
     /**
-     * Response to return.
+     * Take to handle the request and dynamically return the response.
      */
-    private final transient Response origin;
+    private final transient Take origin;
 
     /**
      * Ctor.
      * @param enc Encoding we accept
-     * @param response Response to return
+     * @param take Take to handle the request dynamically.
      */
-    public FkEncoding(final String enc, final Response response) {
```

Comment:
@Happy-Neko same here I think we should leave this constructor for compatibility  and create a new one like `FkContentType` 
@yegor256 what do you think?